### ISSUE TITLE
BigEarthNetV2: Empty Validation Set

### ISSUE THREAD

**Issue by Night-dev1307** [microsoft/torchgeo#2824]
### Description

The problem is:

In the metadata.parquet file for the refined BigEarthNet (=BigEarthNetV2) the split column only holds the values "train", "validation" and "test". The correct key to pass to the c-tor would accordingly be split="validation". However, the c-tor asserts split to be in ["train", "val", "test"] so one cannot use the validation  set.

For additional information on the content of the metadata.parquet refere to: https://bigearth.net/ (Dataset Description Document, page 9) 

### Steps to reproduce

Run the following code:

```
from torchgeo.datasets.bigearthnet import BigEarthNetV2

val_set = BigEarthNetV2(root=DATASET_DIR, bands="s2", split="val")

print(len(val_set))
´´´

This will print: 0

### Version

0.7
